person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
```

Comment:
- [ ] You are giving name a default string but you are doing it wrong :fearful: because only boolean can be given default in an key value format. so your line 5 should look like below :arrow_down:
`def initialize(age, name = 'Unknown', parent_permission: true)`

Reply:
@HAMMAS-SALEEM  thanks fixed